packages/components/src/mobile/html-text-input/container.ios.js

```diff
@@ -1,50 +0,0 @@
-/**
- * External dependencies
- */
-import { UIManager, PanResponder } from 'react-native';
-
-/**
- * WordPress dependencies
- */
-import { Component } from '@wordpress/element';
-
-/**
- * Internal dependencies
- */
-import KeyboardAvoidingView from '../keyboard-avoiding-view';
-import styles from './style.ios.scss';
-
-class HTMLInputContainer extends Component {
-	constructor() {
-		super( ...arguments );
-
-		this.panResponder = PanResponder.create( {
-			onStartShouldSetPanResponderCapture: () => true,
-
-			onPanResponderMove: ( e, gestureState ) => {
-				if ( gestureState.dy > 100 && gestureState.dy < 110 ) {
-					// Keyboard.dismiss() and this.textInput.blur() are not working here
-					// They require to know the currentlyFocusedID under the hood but
-					// during this gesture there's no currentlyFocusedID.
-					UIManager.blur( e.target );
-				}
-			},
-		} );
```

Comment:
AFAIK this logic was used as a workaround for preventing the `TextInput` component from automatically gaining focus when releasing the touch after scrolling. However, this solution no longer works due to the use of the gesture handler library via the `GestureDetector` component ([reference](https://github.com/WordPress/gutenberg/blob/4e5f5f7f00264762b2f612992e710c629460fbc6/packages/components/src/draggable/index.native.js#L77)).